### ISSUE TITLE
Use jstree widget for table of contents

### ISSUE THREAD

**Issue by eliotjordan** [pulibrary/pulfalight#360]
The library we currently use (jqtree) is cannot be navigated easily with a keyboard and does not meet accessibility standards as a result.
